docs/en_US/install/3ds.md

```diff
@@ -83,19 +83,25 @@ Depending on how you installed Nimbus, launch it either through the Homebrew Lau
 
 Your selection persists between reboots.
 
-## Signing into your PNID
+## Signing in into Pretendo Network
 
-The 3DS does not rely on NNIDs for the vast majority of it's game servers. Because of this, using a PNID is also not required for most games<sup><a>[[1]](#footnote-1)</a></sup>.
-
-Setting up a PNID on the 3DS is the same as setting up a NNID. You may either create the PNID on your console, or register from an account [on our website](/account/register) and link it to your console once you're ready.
-
-It is recommended to register the PNID on your device at this time, as registering on the website does not currently allow you to change your user data.
+On your system, go to **System Settings** and click on **Nintendo Network ID Settings**. From there, follow the instructions on screen to create a new Pretendo Network ID or link an existing one created [on our website](/account/register).
 
 <div class="tip red">
 	<strong>CAUTION:</strong>
-	A Pretendo Network ID may not use the same username as the account already linked to your 3DS! Ensure that you have a choose a different name for your PNID than the name on your NNID.
+	A Pretendo Network ID may not use the same username as the account already linked to your 3DS! Ensure that you have chosen a different name for your PNID than the name on your NNID.
```

Comment:
[A user on Discord](https://discord.com/channels/408718485913468928/1067162922989662360/threads/1237976845962576016) suggested the following wording:

> CAUTION: A Pretendo Network ID cannot use the same username as the NNID already linked to your 3DS! Ensure that you choose a different username for your PNID than the username on your NNID.

They said "may not" was vague, which is fair, since "may" often means "optional". "Must not" would be another option. Note how they also said "username" to differentiate from e.g. Mii name.